### PR TITLE
cf-tables cleanup from v0.1.0

### PR DESCRIPTION
**Pull request by mistergone**
This PR and version update reflect cleanup from the initial version 0.1.0, updating cf-tables to **v0.1.1**. After cleanup is merged, we will move to the updates necessary to reach version 1.0.0.
## Changes
- Whitespace fixes
- Reduced nesting of .table__striped
- Various name changes
- Removed .table-cell__right-align from table
- Additional child selectors added
- .table__stack-on-small() removed from .table__entry-header-on-small
- Improved approach to table striping
## Testing
- Simply clone, `npm install`, `bower install`, then `grunt` and be on your way!
- JS tests can be run with `grunt test`
## Review
- @Scotchester, I choose you!
- @jimmynotjim, if you like this kind of thing
## Checklist
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
- [x] Passes all existing automated tests
- [x] New functions include new tests
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged
- [x] Visually tested in supported browsers and devices 
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
## Special Thanks

Thanks to @Scotchester, @sebworks, and @jimmynotjim for putting their eyes on the first pull request - they made it that much easier to do this followup very quickly! I love working with great people who are also great developers! :rocket: :rocket: :rocket:   
## Animated GIF

How not to sort a table:
liz-lemon-table-flip

### CODE REVIEW

File: src/cf-tables.less [hunk 1]
```diff
@@ -335,34 +337,34 @@
...
 
 // Color variables
-@table__head-bg:              #f8f8f8;
-@table__cell-bg:              #ffffff;
-@table__cell-bg_alt:          #f8f8f8;
-@table__scrolling-border:     #e3e4e5;
-@table__border:               #babbbd;
+@table-head-bg:              #f8f8f8;
+@table-cell-bg:              #ffffff;
+@table-cell-bg_alt:           #f8f8f8;
```
Comment (Scotchester) [inline src/cf-tables.less]: Trivial: one extra space in the indentation before the value here.

File: src/cf-tables.less [hunk 2]
```diff
@@ -391,21 +393,30 @@ table {
     }
```
Comment (Scotchester) [inline src/cf-tables.less:394]: `sorted_down` and `sorted_up` need to be changed to use hyphens in the blocks above this line.

File: src/cf-tables.less [hunk 3]
```diff
@@ -335,34 +337,34 @@
...
-        text-align: right;
-    }
+.table_cell__right-align {
+    text-align: right;
 }
 
 .table-wrapper__scrolling {
     box-sizing: border-box;
     overflow-y: hidden;
     table {
```
Comment (Scotchester) [inline src/cf-tables.less:366]: Any reason not to use the child selector here?

**Comment by Scotchester** [comment]
I think I'll create a new issue for this, rather than ask you to address it in this PR, but the color variables should use non-CFPB-specific colors, and we need to add overrides in our `cf-theme-overrides.less` file for the variables to set them to our CFPB -specific values.

**Comment by jimmynotjim** [comment]
:+1: after addressing @Scotchester's comments.

**Comment by Scotchester** [comment]
What's happenin' here?

**Comment by mistergone** [comment]
Sorry, I should've commented before I closed - I was just making for a fresh PR with all issues addressed, rather than tacking onto a PR that had grown whiskers. :grin: New PR is at #12!

**Pull request closed without merging**